toolkit/tools/pkg/imagecustomizerlib/liveosisoimages.go

```diff
@@ -43,13 +46,18 @@ hostonly="no"
 	usrLibLocaleDir = "/usr/lib/locale"
 )
 
+var (
+	kdumpInitramfsRegEx = regexp.MustCompile(`/initramfs-(.*)kdump\.img$`)
+)
+
 type StageFile struct {
 	sourcePath    string
 	targetRelPath string
 	targetName    string
 }
 
-func cleanFullOSFolderForLiveOS(fullOSDir string) error {
+func cleanFullOSFolderForLiveOS(fullOSDir string, kdumpBootFiles *imagecustomizerapi.KdumpBootFilesType,
+	KdumpBootFilesMap map[string]*KdumpBootFiles) error {
```

Comment:
```golang
func cleanFullOSFolderForLiveOS(fullOSDir string, kdumpBootFiles *imagecustomizerapi.KdumpBootFilesType,
	KdumpBootFilesMap map[string]*KdumpBootFiles,
) error {
```

This helps provide visual clarity between the function signature and the first statement.

Reply:
pending